.circleci/scripts/update_config_template.js

```diff
@@ -51,19 +55,54 @@ const main = () => {
               should_install_java: 
                 - true
 `
-  const alteredConfigWithJavaE2es = e2ePackagesWithJava.length > 0 ? 
-                            configTemplate.replace(/# <NEEDS_E2E_TEST_REQUIREMENT_JAVA>/g, '- e2e_tests_with_java')
-                                          .replace(/# <TEST_MATRIX_E2E_JAVA>/g, e2eTestMatrixWithJava)
-                            : configTemplate.replace(/# <NEEDS_E2E_TEST_REQUIREMENT_JAVA>/g, '')
-                                           .replace(/# <TEST_MATRIX_E2E_JAVA>/g, '')
-  const alteredConfigTemplate = e2ePackages.length > 0 ? 
-                alteredConfigWithJavaE2es.replace(/# <NEEDS_E2E_TEST_REQUIREMENT>/g, '- e2e_tests_without_java')
-                            .replace(/# <TEST_MATRIX_E2E>/g, e2eTestMatrix)
-            : alteredConfigWithJavaE2es.replace(/# <NEEDS_E2E_TEST_REQUIREMENT>/g, '')
-                            .replace(/# <TEST_MATRIX_E2E>/g, '')
+  const alteredConfigWithJavaE2es =
+    e2ePackagesWithJava.length > 0
+      ? configTemplate
+          .replace(/# <NEEDS_E2E_TEST_REQUIREMENT_JAVA>/g, '- e2e_tests_with_java')
+          .replace(/# <TEST_MATRIX_E2E_JAVA>/g, e2eTestMatrixWithJava)
+      : configTemplate.replace(/# <NEEDS_E2E_TEST_REQUIREMENT_JAVA>/g, '').replace(/# <TEST_MATRIX_E2E_JAVA>/g, '')
+  const alteredConfigTemplate =
+    e2ePackages.length > 0
+      ? alteredConfigWithJavaE2es
+          .replace(/# <NEEDS_E2E_TEST_REQUIREMENT>/g, '- e2e_tests_without_java')
+          .replace(/# <TEST_MATRIX_E2E>/g, e2eTestMatrix)
+      : alteredConfigWithJavaE2es.replace(/# <NEEDS_E2E_TEST_REQUIREMENT>/g, '').replace(/# <TEST_MATRIX_E2E>/g, '')
+
+  writeFileSync(outputName, alteredConfigTemplate)
+  }
+  
+const generateUtMatrix = (templateName, outputName) => {
+  const configTemplate = readFileSync(templateName, 'utf8')
+  const utPackagesToTest = readFileSync(path.join(__dirname, '..', 'ut_packages_to_test.txt'), 'utf8').split('\n')
+  const utPackagesWithPrettyName = utPackagesToTest.map((pkg) => { return `${pkg.split('/').pop()}:${pkg}` })
   
-  writeFileSync(path.join(__dirname, '..', 'continue_config.yml'), alteredConfigTemplate)
-  console.log(alteredConfigTemplate)
+  // TODO: optimize on the resource_class for each package, using the CI resource utilization analysis
```

Comment:
Can we open a JIRA issue for this?

Reply:
https://salto-io.atlassian.net/browse/SALTO-6217

Reply:
Can you add the issue to the comment so we can easily get back to it?
```suggestion
  // TODO (SALTO-6217): optimize on the resource_class for each package, using the CI resource utilization analysis
```